qemu/tests/watchdog.py

```diff
@@ -282,6 +286,36 @@ def hotplug_unplug_watchdog_device():
                 test.fail("Oops, find watchdog pci, unplug failed")
             logging.info("The WDT remove successfully")
 
+    def watchdog_test_suit():
+        """
+        Run watchdog-test-framework to verify the function of emulated watchdog
+        devices.
+        Test steps of the framework are as follows:
+        1) Set up the watchdog with a 30 second timeout.
+        2) Ping the watchdog for 60 seconds.  During this time the guest should
+        run normally.
+        3) Stop pinging the watchdog and just count up.  If the virtual watchdog
+        device is set correctly, then the watchdog action (eg. pause) should
+        happen around the 30 second mark.
+        """
+
+        _watchdog_device_check(test, session, watchdog_device_type)
+        watchdog_test_lib = params["watchdog_test_lib"]
+        src_path = os.path.join(data_dir.get_deps_dir(), watchdog_test_lib)
+        vm.copy_files_to(src_path, "/home")
+        try:
+            session.cmd_output("/home/%s --yes" % os.path.basename(watchdog_test_lib),
+                               timeout=130)
```

Comment:
@YanhuiMa  As watchdog_test_suit can test anyone watchdog action. So, this case should test all actions(e.g reset,shutdown,debug,.....).  Currently, It only cover pause action.

Reply:
@xiangchunfu I think the purpose of the case is to ensure watchdog works well, no need to cover all actions in the case. We have dedicated cases for all actions.